extensions/amp-story-panning-media/README.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
If this is a component documentation (in amp.dev/documentation/components), you'd want this file to be named after the extension: `amp-story-panning-media.md`. If this is just a README for the preview and don't want it as component documentation (meaning you plan to upload a documentation .md later on), all these headers are unnecessary